google/resource_google_folder.go

```diff
@@ -0,0 +1,166 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	resourceManagerV2Beta1 "google.golang.org/api/cloudresourcemanager/v2beta1"
+	"strings"
+)
+
+func resourceGoogleFolder() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGoogleFolderCreate,
+		Read:   resourceGoogleFolderRead,
+		Update: resourceGoogleFolderUpdate,
+		Delete: resourceGoogleFolderDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceGoogleFolderImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			// Format is 'folders/{folder_id}.
+			// The terraform id holds the same value.
+			"name": &schema.Schema{
```

Comment:
The convention we've mostly been falling into is to list all Required fields first, then Optional, then Computed. I think that would make it more clear that display_name is what the user sets, rather than name (since for so many other resources it's name)

Reply:
Done